internal/rootcoord/import_manager.go

```diff
@@ -976,22 +983,29 @@ func (m *importManager) listAllTasks(colName string, limit int64) []*milvuspb.Ge
 		log.Error("failed to load from task store", zap.Error(err))
 		return tasks
 	}
-	taskCount := int64(0)
+
+	// filter tasks by collection name
+	// TODO: how to handle duplicated collection name? for example: a new collection has same name with a dropped collection
```

Comment:
The task meta shall store collection id instead of collection name. It's suggested to convert collection name to id once the request is inside `rootcoord`